src/db/schema.ts

```diff
@@ -50,19 +53,40 @@ export const feelingEnum = pgEnum("feeling", [
   "Sad",
   "I don't know",
 ]);
+export const genderEnum = pgEnum("gender", ["Male", "Female", "Other"]);
+export const ageEnum = pgEnum("age", [
+  "18-24",
+  "25-34",
+  "35-44",
+  "45-54",
+  "55-64",
+  "65+",
+]);
+export const specialtyEnum = pgEnum("specialty", [
+  "Anxiety",
+  "Depression",
+  "PTSD",
+  "Bipolar Disorder",
+  "Obsessive-Compulsive Disorder",
+  "Stress Management",
+  "Anger Management",
+]);
```

Comment:
These are just mental health disorders, we need specialties like addiction, family, health, etc. (find more dont just use these)